app/config/config.yml

```diff
@@ -192,6 +198,21 @@ doctrine:
 doctrine_migrations:
     dir_name: "%kernel.root_dir%/DoctrineMigrations"
 
+# Swiftmailer Configuration
+swiftmailer:
+    spool:     { type: memory }
+    transport: "%mailer_transport%"
+    host:      "%mailer_host%"
+    username:  "%mailer_user%"
+    password:  "%mailer_password%"
+    # using an env var throws an error
+    # auth_mode: "%mailer_auth_mode%"
+    # auth_mode: "%env(MAILER_AUTH_MODE)%"
+    auth_mode: cram-md5
```

Comment:
If we can figure it out, we should make this configurable via an environment variable.

Reply:
I wasn't able to find a fix for this. We might want to make this an issue to solve later.